components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2414,13 +2414,13 @@ void RewardsServiceImpl::HandleFlags(const std::string& options) {
       continue;
     }
 
+    // The "persist-logs" command-line flag is deprecated and will be removed
+    // in a future version. Use --enable-features=BraveRewardsVerboseLogging
+    // instead.
     if (name == "persist-logs") {
       const std::string lower = base::ToLowerASCII(value);
-
       if (lower == "true" || lower == "1") {
-        should_persist_logs_ = true;
-      } else {
-        should_persist_logs_ = false;
+        persist_log_level_ = kDiagnosticLogMaxVerboseLevel;
```

Comment:
There would appear to be no way to set to a higher verbosity. `kDiagnosticLogMaxVerboseLevel` is set to 6. But ads and ledger can log up to any level.

Reply:
Currently, AFAIK we don't have a way to set the log level to a specific number at the command line. I thought about adding that as a feature but decided it was out of scope for this PR.

Reply:
ah yes :-) but def a good idea for a future update